hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1232,11 +1253,11 @@ public ListKeysResult listKeys(String volumeName, String bucketName,
         cacheKeyMap.put(key, omKeyInfo);
       }
     }
-
+    long startNano, stopNano = 0;
```

Comment:
Can you please rename the variables to clarify the difference between `startNanos` and `startNano`?

Reply:
+1

Reply:
Sure @adoroszlai, renamed variables for clarification.